docs/source/contrib_guide_code.rst

```diff
@@ -0,0 +1,106 @@
+reside and leave a comment in the file describing what issue you are trying
+to address.
+
+5. Open a pull request to the repository with [WIP] appended to the front
+so that the core team is aware that you are actively pursuing the issue.
+When creating a pull request, make sure that the title clearly and concisely
+described what your code does. For example, we might use the title "Updated
+error message on ExampleException". In the body of the pull request, make sure 
+that you include the phrase "Closes #issue-number-here", where the issue number is
+the issue number of the issue that you are addressing in this PR.
+
+6. Run the test suite locally in order to ensure that everything is properly
+configured on your system. Refer to the repository's README for information
+on how to run the test suite. This will typically require that you run the
+``nosetests`` command on the commandline.
+
+7. Find the test file associated with the module that you will be changing. 
+In the test file, add some tests that outline what you expect the behavior 
+of the change should be. If we continue with our example of updating the 
+text that is logged on error, we might write test cases that check to see 
+if the exception raised when you induce the error contains the appropriate 
+string. When writing test cases, make sure that you test for the following 
+things.
+
+* What is the simplest test case I can write for this issue?
+* What will happen if your code is given messy inputs?
+* What will happen if your code is given no inputs?
+* What will happen if your code is given too few inputs?
+* What will happen if your code is given too many inputs?
+  
+If you need assistance writing test cases, you can place a comment on the
+pull request that was opened earlier and one of the core team members will
+be able to help you.
+
+8. Go back to the file that you are updating and begin adding the code for your
+pull request.
+
+9. Run the test suite again to see if your changes have caused any of the test
+cases to pass. If any of the test cases have failed, go back to your code and 
+make the updates necessary to have them pass.
+
+10. Once all of your test cases have passed, commit both the test cases and the
+updated module and push the updates to the branch on your forked repository.
+
+11. Once you are ready for your pull request to be reviewed, remove the [WIP] tag 
+from the front of issue, a project reviewer will review your code for quality. 
+You can expect the reviewer to check for the documentation provided in the changes 
+you made, how thorough the test cases you provided are, and how efficient your 
+code is. Your reviewer will provide feedback on your code and you will 
+have the chance to edit your code and apply fixes.
+
+12. Once your PR is ready to become a part of the code base, it will be merged
+by a member of the core team.
+
+Contribution Workflow
+----------------------
+.. image:: http://i.imgur.com/J0Q5H7s.png
```

Comment:
@captainsafia Let's make an ``_images`` directory and drop this png image file into the ``docs/source/_images/``. Update the link to grab from ``_images`` directory and Sphinx will be happier and not throw a warning. Your approach is fine but it's such a cool graphic let's make sure that we have it easily accessible :smiley_cat: